docs/source/BuildingRunningTesting/BuildRunLandDA.rst

```diff
@@ -0,0 +1,266 @@
+
+.. attention::
+   
+   These steps are designed for use on :ref:`Level 1 <LevelsOfSupport>` systems (i.e., Hera and Orion) and may require significant changes on other systems. It is recommended that users on other systems run the containerized version of Land DA. Users may reference :numref:`Chapter %s: Containerized Land DA Workflow <Container>` for instructions. 
+
+Create a Working Directory
+*****************************
+
+Create a directory for the Land DA experiment (``$LANDDAROOT``):
+
+.. code-block:: console
+
+   mkdir /path/to/landda
+   cd /path/to/landda
+   export LANDDAROOT=`pwd`
+
+where ``/path/to/landda`` is the path to the directory where the user plans to run Land DA experiments. 
+
+.. _GetData:
+
+Get Data
+***********
+
+:numref:`Table %s <Level1Data>` shows the locations of pre-staged data on NOAA :term:`RDHPCS` (i.e., Hera and Orion). 
+   
+.. _Level1Data:
+
+.. table:: Level 1 RDHPCS Data
+
+   +-----------+--------------------------------------------------+
+   | Platform  | Data Location                                    |
+   +===========+==================================================+
+   | Hera      | /scratch2/NAGAPE/epic/UFS_Land-DA/inputs         |
+   +-----------+--------------------------------------------------+
+   | Orion     | /work/noaa/epic/UFS_Land-DA/inputs               |
+   +-----------+--------------------------------------------------+
+
+Users can either set the ``LANDDA_INPUTS`` environment variable to the location of their system's pre-staged data or use a soft link to the data. For example, on Hera, users may set: 
+
+.. code-block:: console
+
+   export LANDDA_INPUTS=/scratch2/NAGAPE/epic/UFS_Land-DA/inputs
+
+Alternatively, users can add a soft link to the data. For example, on Orion:
+
+.. code-block:: console
+
+   cd $LANDDAROOT
+   ln -fs /work/noaa/epic/UFS_Land-DA/inputs
+
+Users who have difficulty accessing the data on Hera or Orion may download it according to the instructions in :numref:`Section %s <GetDataC>`. Users with access to data for additional experiments may use the same process described above to point or link to that data by modifying the path to the data appropriately. 
+
+Users who are not using Land DA on Hera or Orion should view :numref:`Chapter %s <Container>` for instructions on running the containerized version of Land DA. :numref:`Section %s <GetDataC>` explains options for downloading the sample data onto their system. 
+
+Get Code
+***********
+
+Clone the Land DA repository. To clone the ``develop`` branch, run: 
+
+.. code-block:: console
+
+   git clone -b develop --recursive https://github.com/ufs-community/land-DA_workflow.git
+
+To clone the most recent release, run the same command with |tag| in place of ``develop``: 
+
+.. code-block:: console
+
+   git clone -b ufs-land-da-v1.2.0 --recursive https://github.com/ufs-community/land-DA_workflow.git
```

Comment:
@gspetro-NOAA please change the branch name from 'ufs-land-da-v1.2.0' to 'release/public-v1.2.0'.

Reply:
@rhaesung Are we planning to create a branch? Your change will only work for a link if we create a branch with that name, but I thought we were just using a release tag off of `develop`.

Reply:
Ok, if we create the branch, then I can change this and the README.md link to `/release/public-v1.2.0`.